app/js/main.js

```diff
@@ -287,6 +334,11 @@ function setupMeteringDemo(subscriptions) {
             MeteringDemo.openPaywall();
           });
         } else {
+          // The publisher must inform showcase of the 'paywall' event
+          subscriptions.setShowcaseEntitlement({
+            isUserRegistered: true,
+            entitlement: ShowcaseEntitlementEvent.IMPRESSION_PAYWALL,
+          });
```

Comment:
Could you add a newline here? I noticed the code stanzas in the `setupMeteringDemo` method have newlines between them. This new stanza doesn't have a newline between it and the next stanza though